HW16/pageobjects/CartPage.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+from HW16.pageobjects.BasePage import BasePage
+from HW16.utils.PropertiesHandler import get_properties
+from selenium.webdriver.remote.webdriver import WebDriver
+
+LOGGER = logging.getLogger()
+
+
+class CartPage(BasePage):
```

Comment:
Page object should be a representation of the page and shouldn't contain asserting methods